lib/intercom/request.rb

```diff
@@ -58,6 +58,7 @@ def client(uri)
     end
 
     def execute(target_base_url=nil, username:, secret: nil)
+      retries ||= 3
```

Comment:
this is outside of `try` block. we can just do `retries = 3`

Reply:
https://stripe.com/docs/subscriptions/invoices#invoice-states